field/src/exponentiation.rs

```diff
@@ -41,7 +41,7 @@ pub fn exp_1717986917<AF: AbstractField>(val: AF) -> AF {
 
 pub fn exp_1420470955<AF: AbstractField>(val: AF) -> AF {
     // Note that 3 * 1420470955 = 2*(2^31 - 2^24) + 1 = 1 mod (p - 1).
-    // Thus as a^{p - 1} = 1 for all a \in F_p, (a^{1725656503})^7 = a.
```

Comment:
I think there might be 2 typo's here. In addition to the one you fixed, the ^7 should be a ^3.

Thanks for catching it.